src/css/App.css

```diff
@@ -4,43 +4,193 @@ a {
 
 .centre-text {
   text-align: center;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2rem;
+}
+
+.content {
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.content a {
+  color: #fff;
+}
+
+.content a:hover {
+  color: #fd79a8;
+}
+
+.left-text {
+  text-align: left;
+  width: 80% !important;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2rem;
+}
+
+.gap-div {
+  height: 4rem;
 }
 
 .Menu a {
   color: #fff;
+  text-align: right !important;
```

Comment:
do we need the `!important` tags in here?

Reply:
I only added the important tag when it was required to override default styling from react-bootstrap, so yes probably. I wouldn't add it if it wasn't necessary. 
I also didn't want to go through and undo every one of them to double check. You are welcome to if you want.